tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -45,7 +60,7 @@ type AwsProvider struct {
 	// instanceType is the flavor of VM to be used
 	instanceType string
 	// sshKey is the ssh key to access the VM created. Please note that key should be uploaded to AWS before
-	// using this flag
+	// using this flag. AWS encrypts the password of the Windows instance created with this public key
```

Comment:
i don't understand the `using this flag.` part of this comment

Reply:
If you use the this sshKey and the key is not present in the AWS list of keys, it'd fail